frontend/app/src/App.tsx

```diff
@@ -1159,7 +1175,7 @@ export class App extends PureComponent<Props, State> {
   /**
    * Performs one-time initialization. This is called from `handleNewSession`.
```

Comment:
nit: maybe also adapt the comment that this initialization can happen on the first connection or on a reconnection.